docs/INSTALL.md

```diff
@@ -25,31 +17,22 @@ Templates and other build generated files are hosted on an S3 bucket by project
 These templates are ready to use to install the latest release of MSAM.
 
 ```
-https://rodeolabz-us-west-2.s3.amazonaws.com/aws-media-services-application-mapper/latest/aws-media-services-application-mapper-release.template
-https://rodeolabz-us-west-2.s3.amazonaws.com/aws-media-services-application-mapper/latest/msam-browser-app-release.template
-https://rodeolabz-us-west-2.s3.amazonaws.com/aws-media-services-application-mapper/latest/msam-core-release.template
-https://rodeolabz-us-west-2.s3.amazonaws.com/aws-media-services-application-mapper/latest/msam-dynamodb-release.template
-https://rodeolabz-us-west-2.s3.amazonaws.com/aws-media-services-application-mapper/latest/msam-events-release.template
-https://rodeolabz-us-west-2.s3.amazonaws.com/aws-media-services-application-mapper/latest/msam-iam-roles-release.template
+https://solutions-reference.s3.amazonaws.com/aws-media-services-application-mapper/latest/aws-media-services-application-mapper-release.template
```

Comment:
The way we are building, I think only the main template ends up in the main solutions-reference bucket. The rest of the templates will be in regional buckets. Might need to review VWR to compare. Then update the rest of the links on this README as needed.

Reply:
I could very well be remembering this incorrectly :)

Reply:
We place templates in global during the build and copy *.template to regional at the end of build_s3_dist.

```
# copy all processed templates to the regional assets directory
cp *.template $build_dist_dir
```